cli/src/main/java/org/infinispan/cli/commands/rest/Migrate.java

```diff
@@ -111,6 +111,27 @@ public boolean isHelp() {
       }
    }
 
+
+   @CommandDefinition(name = "source-connection", description = "Obtains the remote store configuration if a cache was previously connected to another cluster")
+   public static class ClusterSourceConnection extends RestCliCommand {
```

Comment:
In what scenarios do you envisage a user requiring this command via the CLI?

I understand that it's useful from an Operator perspective, but do we need to expose it via the CLI if we're not providing methods for all of the `rolling-upgrade` CRUD operations?

Reply:
The CLI must be capable of orchestrate a full rolling upgrade. This call is for troubleshooting purposes.
Regarding CRUD, the cli does expose them: ```connect```, ```disconnect```, maybe I should align the names with the REST methods

Reply:
> Regarding CRUD, the cli does expose them: `connect`, `disconnect`, maybe I should align the names with the REST methods

I was more meaning that we now don't expose the check to see if a connection has been established. I think the existing CLI names are fine.

I don't have strong views about having this method in the CLI. Thinking about it again, I would say the more diagnostic tools the better when the overhead is this low.

Reply:
> I was more meaning that we now don't expose the check to see if a connection has been established.

In the CLI, the ```source-connection``` will return a text ```Not Found:``` when there is no connection, so it can be used to check if there is connection :)

I had issue implementing ```is-connected``` in the CLI since it's text output oriented and a head request does not return any text in the RestResponse. I will double check to see if there is a way

Reply:
Yeah, basically the CLI command can either print the body, the headers, do nothing or return a file. So I will leave aside the implementation of is-connected and users can leverage get-connection instead